e2e/nomostest/new.go

```diff
@@ -269,6 +274,14 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 	// init the Client & WatchClient
 	nt.RenewClient()
 
+	// Create workload to disable scale-to-zero.
+	// Scale-to-zero breaks API discovery on GKE Autopilot: b/209800496S
```

Comment:
Could you add a bit more context on the commit message for why we need this? e.g. what problems does this cause for the e2e tests

Reply:
done